### ISSUE TITLE
Package seems to be ignoring validateloginattempt

### ISSUE THREAD

**Issue by mitar** [sandstorm-io/meteor-accounts-sandstorm#22]
http://docs.meteor.com/#/full/accounts_validateloginattempt

Even if callback is returning `false`, user is still logged in.

**Comment by mitar** [comment]
In short, callback is not called.

**Comment by kentonv** [comment]
This does appear to be a bug.

That said, what is the use case for this on Sandstorm?

**Comment by mitar** [comment]
For me, security: https://github.com/mitar/council-app/blob/master/packages/core/accounts.coffee#L14

**Comment by kentonv** [comment]
OK, so you're trying to disable non-Sandstorm login mechanisms when Sandstorm is enabled.

I actually think we should build this into accounts-sandstorm. Traditionally we have told people that they should build with a different package set when building for Sandstorm vs. outside, but we've noticed some projects forget to do this. Perhaps accounts-sandstorm should automatically install a login validator function that rejects everything. Since we now explicitly check for Sandstorm, it's safe to do this.

**Comment by mitar** [comment]
Yes. I think quite some functionalities which I had to implement in my app should be moved out into some Meteor-Sandstorm integration.

But this is just my use case. I can imagine people use `validateLoginAttempt` to log login attempts.

**Comment by kentonv** [comment]
OK, I implemented "highlander mode" and published version 0.3.0, so you can remove your login validator.

I looked briefly at how to call the login validator callback, but this seems awkward. accounts-sandstorm doesn't actually use the Meteor Accounts framework as it isn't a great fit, and the validation hooks are buried pretty deep in there.

I'm open to a pull request but given that there's seemingly no immediate need I'm not going to work on this any further for now.

**Comment by mitar** [comment]
So cool!

**Comment by mitar** [comment]
For the same reason, this also does not call: http://docs.meteor.com/#/full/accounts_onlogin

**Comment by mitar** [comment]
Pull request done.

While working on this I discovered that more importantly onLogin and onLoginFailure hooks were also not called.

This now calls all that and nicely integrates with Meteor accounts.
